Component/Text Utilities/textUtilities.html

```diff
@@ -0,0 +1,208 @@
+          </li>
+          <li>
+            <a href="#alert" class="sidebar-menu" rel="noopener noreferrer">
+              Text Utilities
+            </a>
+          </li>
+          <li>
+            <a href="#alert" class="sidebar-menu" rel="noopener noreferrer">
+              List
+            </a>
+          </li>
+          <li>
+            <a href="#alert" class="sidebar-menu" rel="noopener noreferrer">
+              Navigation
+            </a>
+          </li>
+          <li>
+            <a href="#alert" class="sidebar-menu" rel="noopener noreferrer">
+              Modal
+            </a>
+          </li>
+          <li>
+            <a href="#alert" class="sidebar-menu" rel="noopener noreferrer">
+              Rating
+            </a>
+          </li>
+          <li>
+            <a href="#alert" class="sidebar-menu" rel="noopener noreferrer">
+              Toast
+            </a>
+          </li>
+          <li>
+            <a href="#alert" class="sidebar-menu" rel="noopener noreferrer">
+              Simplified Grid
+            </a>
+          </li>
+        </ul>
+      </div>
+      <div class="main">
+        <h1 class="font-xl margin-m2">Typography</h1>
+        <br />
+        <hr />
+        <h3 class="font-medium margin-m2">Headings</h3>
+        <div class="box">
+          <div class="h1">H1-Heading</div>
+          <br />
+          <div class="h2">H2-Heading</div>
+          <br />
+          <div class="h3">H3-Heading</div>
+          <br />
+          <div class="h4">H4-Heading</div>
+          <br />
+          <div class="h5">H5-Heading</div>
+          <br />
+          <div class="h6">H6-Heading</div>
+          <br />
+        </div>
+        <div class="margin-m2 padding-p2">
+          <code class="code-container container-flex flex-center">
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%28231%2C229%2C229%2C1%29&t=one-light&wt=none&l=text%2Fx-csrc&width=471.5&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=143%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522h1%2522%253EH1-Heading%253C%252Fdiv%253E%2520%250A%253Cdiv%2520class%253D%2522h2%2522%253EH2-Heading%253C%252Fdiv%253E%2520%250A%253Cdiv%2520class%253D%2522h3%2522%253EH3-Heading%253C%252Fdiv%253E%2520%250A%253Cdiv%2520class%253D%2522h4%2522%253EH4-Heading%253C%252Fdiv%253E%2520%250A%253Cdiv%2520class%253D%2522h5%2522%253EH5-Heading%253C%252Fdiv%253E%2520%250A%253Cdiv%2520class%253D%2522h6%2522%253EH6-Heading%253C%252Fdiv%253E%2520"
+              style="
+                width: 100%;
+                height: 178px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
+          </code>
+        </div>
+        <h3 class="font-medium margin-m2">Paragraphs</h3>
+
+        <div class="box">

```

Comment:
Remove extra spacing to make the code more cleaner.

Reply:
yeah some time prettier doesnt work properly